### ISSUE TITLE
Smoke test voice interface

### ISSUE THREAD

**Issue by okjuan** [okjuan/muze#31]

**Comment by okjuan** [comment]
After some fiddling, I came up with this cURL request that pings my Dialogflow agent:
```
$ curl -X POST -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) -H "Content-Type: application/json; charset=utf-8" \
        "https://dialogflow.googleapis.com/v2/projects/muze-2b5fa/agent/sessions/123:detectIntent" \
        --data "{ \
                'queryInput': { \
                        'audioConfig': { \
                                'audioEncoding': 'AUDIO_ENCODING_LINEAR_16', \
                                'sampleRateHertz': 22050, \
                                'languageCode': 'en' \
                        }\
                },\
                'inputAudio': '"$(say -o play_song.wav  --data-format=LEI16@22050 "Play My Boo"; base64 -i play_song.wav)"' \
        }"
```

When the Muze service is not running, the response looks like:
```
{
  "responseId": "20252fb6-787a-4df5-a373-a905ff597284",
  "queryResult": {
    "queryText": "Play My Boo",
    "speechRecognitionConfidence": 0.95631635,
    "parameters": {
      "song": "My Boo",
      "artist": ""
    },
    "allRequiredParamsPresent": true,
    "fulfillmentText": "My Boo is by Ariana Grande... or someone else. I'm not sure yet :^(",
    "fulfillmentMessages": [
      {
        "text": {
          "text": [
            "My Boo is by Ariana Grande... or someone else. I'm not sure yet :^("
          ]
        }
      }
    ],
    "intent": {
      "name": "projects/muze-2b5fa/agent/intents/f552fc98-601c-4cc3-b890-4135a727e8f7",
      "displayName": "Find Song"
    },
    "intentDetectionConfidence": 1,
    "diagnosticInfo": {
      "webhook_latency_ms": 32
    },
    "languageCode": "en"
  },
  "webhookStatus": {
    "code": 5,
    "message": "Webhook call failed. Error: 404 Not Found"
  }
}
```

So we can see that the intent was matched with high confidence :)

I used some stuff online to figure it out:
* [SO Answer](https://stackoverflow.com/questions/9729153/error-on-say-when-output-format-is-wave) about `say`
* [Dialogflow DetectIntent Endpoint Docs](https://cloud.google.com/dialogflow-enterprise/docs/reference/rest/v2/projects.agent.sessions/detectIntent#QueryResult)

**Comment by okjuan** [comment]
Resolved by branch `test_voice_agent`(see 77a000a)
